index.d.ts

```diff
@@ -262,6 +262,11 @@ declare namespace execa {
 		The signal that was used to terminate the process.
 		*/
 		signal?: string;
+
+		/**
+		Original error message when the child process exits due to an error or a timeout.
```

Comment:
Can you describe here how it's different from `.message`? (and in the readme)

Also in what scenarios it's `undefined`.

Reply:
Done in 7290b6bdccd68f7327fa74d29a222fe60aecc91a